presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -3364,9 +3364,7 @@ public void testCorrelatedScalarSubqueries()
         assertQuery("SELECT (SELECT count(*) FROM (VALUES (7,1)) t(orderkey, value) WHERE orderkey = corr_key LIMIT 1) FROM (values 7) t(corr_key)");
 
         // explicit LIMIT in subquery
-        assertQueryFails(
-                "SELECT (SELECT count(*) FROM (VALUES (7,1)) t(orderkey, value) WHERE orderkey = corr_key GROUP BY value LIMIT 1) FROM (values 7) t(corr_key)",
-                "line 1:9: Given correlated subquery is not supported");
```

Comment:
Why this is removed?

Reply:
When limit is removed (because we estimate aggregation cardinality better) then we are left with scalar aggregation subquery, which we support better.

I've added another test case that still fails